packages/frontend/src/constants/index.ts

```diff
@@ -70,6 +70,9 @@ export const BLOCKED_COUNTRIES = [
 ]
 export const ALLOW_WITHDRAWALS = ['CA']
 
+export const WHITELISTED_DATACENTER_ORGANIZATIONS = ['amazon', 'google', 'azure', 'oracle', 'softlayer', 'cloudflare']
```

Comment:
should this be ENV for easier adjustment, etc?

Reply:
agreed

Reply:
done: https://github.com/opynfinance/squeeth-monorepo/pull/1012/commits/e075f27bd58e58de5711df971f55114375abfa1a